### ISSUE TITLE
Update of existing mosaicjson does not update or clear cache

### ISSUE THREAD

**Issue by philvarner** [developmentseed/cogeo-mosaic#176]
The backend method read/get methods are wrapped in a cachetools ttl cache. Updating an existing mosaic will write this new value to the backend, but it won't be available to read for up to 5 minutes (default). 

This is a complex behavior to implement correctly, but the existing implementation is confusing to users, especially because there is no documentation that this is the behavior. I would expect and update and the read to return what I just updated, not the old version until it expires.

**Comment by kylebarron** [comment]
Interesting. So we should be able to call `cache.clear()` from inside the update method? https://stackoverflow.com/a/59443455

**Comment by philvarner** [comment]
> Interesting. So we should be able to call `cache.clear()` from inside the update method? https://stackoverflow.com/a/59443455

Yes, but you can also clear just one entry with `pop`, which is probably what should be done. It would probably get more complicated with the quadkeys, but it seems like there's a solution there.

This doc https://cachetools.readthedocs.io/en/stable/ has an example if you search for `To perform operations on the cache object, for example to clear the cache during runtime`
